### PR TITLE
TTP's for Defense Evasion and Credential Access

### PR DESCRIPTION
**Pull request by Abyss101**
1. Disable Windows EventLog via MinNt Entry via Reg Command
2. Disable Windows EventLog via EventCleaner
3. Use of NTDS for Copying DC Database

### CODE REVIEW

File: defense-evasion/8feb27ce-63db-472f-b8e9-038134e9d79a.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+id: 8feb27ce-63db-472f-b8e9-038134e9d79a
+metadata:
+  version: 1
+  authors:
+    - Archie Pocsedio
+  tags: destructive
```
Comment (khyberspache) [inline defense-evasion/8feb27ce-63db-472f-b8e9-038134e9d79a.yml:6]: needs to be a list:

```yaml
tags:
  - destructive
```

File: defense-evasion/e4209652-91f4-4c7b-8b5d-db246bb08764.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+id: e4209652-91f4-4c7b-8b5d-db246bb08764
+metadata:
+  version: 1
+  authors:
+    - Archie Pocsedio
+  tags: destructive
```
Comment (khyberspache) [inline defense-evasion/e4209652-91f4-4c7b-8b5d-db246bb08764.yml:6]: needs to be a list:

```yaml
tags:
  - destructive
```

**Review by khyberspache — CHANGES_REQUESTED** [review]
Very small tweaks. Excellent work!

File: defense-evasion/8feb27ce-63db-472f-b8e9-038134e9d79a.yml (hunk 1 above)
Comment (Abyss101) [reply defense-evasion/8feb27ce-63db-472f-b8e9-038134e9d79a.yml]: Hello, done revision. 

Thank you

File: defense-evasion/e4209652-91f4-4c7b-8b5d-db246bb08764.yml (hunk 2 above)
Comment (Abyss101) [reply defense-evasion/e4209652-91f4-4c7b-8b5d-db246bb08764.yml]: Hello, done revision. 

Thank you

**Pull request merged**